rotkehlchen/api/server.py

```diff
@@ -290,6 +291,7 @@
     ('/assets/prices/historical', HistoricalAssetsPriceResource),
     ('/assets/ignored', IgnoredAssetsResource),
     ('/assets/ignored/whitelist', FalsePositiveSpamTokenResource),
+    ('/assets/spam/', SpamTokenResource),
```

Comment:
Since this is at least for now only EVM tokens perhaps do
```suggestion
    ('/assets/evm/spam/', SpamEvmTokenResource),
```